tokens/font.jsonc

```diff
@@ -3,14 +3,14 @@
 		"family": {
 			// These first two ones are the only two that are necessary tokens, the three at the end are only relevant for tailwind
 			"base": {
-				"value": "'{asset.font.db-screensans-regular.fontfamily.value}', Helvetica, Arial, sans-serif"
+				"value": "'DBScreenSans', Helvetica, Arial, sans-serif"
 			},
 			"headline": {
-				"value": "'{asset.font.db-screenhead.fontfamily.value}', Helvetica, Arial, sans-serif"
+				"value": "'DBScreenHead', Helvetica, Arial, sans-serif"
```

Comment:
These values were actually correct previously and are now incorrect.

Reply:
For now, I changed it to test the build process. We should ask UX here as well how we manage this.